src/test/javascript/portal/cart/DownloadPanelSpec.js

```diff
@@ -234,6 +235,17 @@ describe("Portal.cart.DownloadPanel", function() {
         });
     });
 
+    describe('getEmbededTitle', function() {
+
+        it('Extracts the real title super trickerly embeded title in the description field from Geonetwork', function() {
```

Comment:
Your right. Its a bit weird. For the moment we are limited as too what we can configure in Geonetwork. He's the last email on the subject from @pblain 

"Thanks Rog.  We will go with the brackets idea then.  
For everyone else's benefit, here's why I don't like it:

1. We will need to ask AODN contributors to conform to a rather peculiar standard if they want multi-WFS.  
2. If AODN contributors aren't aware of the "standard" they may innocently use brackets in their metadata for some other reason and get unexpected results
3. The long-winded description is redundant:  Users who know what a WFS is don't need it.  Users who don't know what a WFS won't be helped by it.
4. More people use the portal than GeoNetwork so the portal should be our priority.  
5. It's an inelegant and unwieldy solution - which means an increased chance of unanticipated problems.
Cheers,
Pete."